### PR TITLE
Shorten long strakids

### LINKED ISSUE #229: 🐛 [UI BUG] Long Stark IDs break Navbar UI
### Description

**Description:**
I've noticed that when a user has a particularly long Stark ID, the UI for the navbar gets distorted and breaks. This affects the overall user experience and the readability of the ID.

**Steps to Reproduce:**
1. Log in with a user that has an extended Stark ID.
2. Observe the navbar and how the ID gets displayed.

**Expected Result:**
The navbar should gracefully handle Stark IDs of any length without breaking the UI.

**Actual Result:**
The navbar UI gets disrupted with long Stark IDs.

**Screenshots:**
image
image

**Possible Solution:**
I'd like to propose two potential solutions:
1. Implement a max length for displaying Stark IDs on the navbar, and shorten them (e.g., using ellipses in the middle) if they exceed this length.
2. Introduce an animation where the Stark ID continuously moves from left to right, within a fixed-size container, so the user can still read the entire ID over time.

I'm open to contributing a PR to address this issue. However, I'd like to get some guidance on the preferred solution and, if we're shortening the IDs, what the maximum display length should be.

Thank you for considering this issue.

### PR DESCRIPTION
**Pull request by iamoskvin**
Fixes #229

### CODE REVIEW

File: src/components/Web3Status/index.tsx [hunk 1]
```diff
@@ -194,6 +194,7 @@ function Web3StatusInner({ starkID }: { starkID?: string }) {
   const { t } = useTranslation()
   const { error } = useStarknetReact()
   const { address, connector } = useAccountDetails()
+  starkID = 'lxhgldfsflgldthtlxhgldfsflgldthtlxhgldfsflgldthtlxhgldfsflgldthtlxhgldfsflgldtht'
```
[the remark this answers is not in the document]
Comment (iamoskvin) [reply src/components/Web3Status/index.tsx:197]: oh, sorry, my bad :(
I don't have other way for testing, besides hardcoding starkID and I forgot to remove this line

File: src/utils/index.ts [hunk 2]
```diff
@@ -81,6 +81,13 @@ export function shortenAddress(address: string, chars = 4): string {
   return `${parsed.substring(0, chars + 2)}...${parsed.substring(63 - chars)}`
 }
 
+export function shortenStarkID(starkID: string): string {
+  if (starkID.length <= 21) {
```
[the remark this answers is not in the document]
Comment (iamoskvin) [reply src/utils/index.ts:88]: done.

File: src/components/Web3Status/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (iamoskvin) [reply src/components/Web3Status/index.tsx]: I have removed it

**Review by retroboydev — COMMENTED** [review]
please also check the `WalletModal` component, it also displays starknet id (see `ENSName` prop)

**Comment by iamoskvin** [comment]
> please also check the `WalletModal` component, it also displays starknet id (see `ENSName` prop)



> please also check the `WalletModal` component, it also displays starknet id (see `ENSName` prop)

done

**Pull request merged**